src/test/suite/gettingStarted.test.ts

```diff
@@ -0,0 +1,42 @@
+import * as assert from "assert";
+import { afterEach } from "mocha";
+import * as sinon from "sinon";
+import { onPluginInstalledEmitter } from "../../events/onPluginInstalledEmitter";
+import { getContext } from "./utils/preReleaseInstaller.utils";
+import handlePluginInstalled from "../../handlePluginInstalled";
+import { ALREADY_OPENED_GETTING_STARTED_KEY } from "../../state/gettingStartedOpenedState";
+import * as gettingStartedWebview from "../../webview/openGettingStartedWebview";
+
+suite("Getting started tests", () => {
+  afterEach(() => {
+    sinon.verifyAndRestore();
+  });
+
+  test("Should open the webview if the user has already seen it", () => {
```

Comment:
i think you confused the test names - should be "Should **not** open" if the user already seen it, and "Should ~not~ open" in the other test

Reply:
lol you're right. thanks! :)